recipes/analisi/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir build_serial
+cd build_serial
+cmake ../ -DBUILD_TESTS=OFF -DPYTHON_EXECUTABLE="$PYTHON"
+make
+cp -v analisi "$PREFIX/bin/analisi_serial"
+cp -v pyanalisi*.so "$SP_DIR/pyanalisi.so"
+
+cd ../
+mkdir build_mpi
+cd build_mpi
+cmake ../ -DCMAKE_CXX_COMPILER=mpicxx -DCMAKE_C_COMPILER=mpicc -DUSE_MPI=ON -DBUILD_TESTS=OFF -DPYTHON_EXECUTABLE="$PYTHON"
+make
```

Comment:
Any `make tests` or `make check` we can run on these two install commands.

Also, I would consifer splitting the mpi and serial packages.

Reply:
Added test run.
I don't think that splitting in two different packages the mpi and the thread only version is something that I want to do at the moment.